webapp/components/profile_picture.jsx

```diff
@@ -25,21 +28,96 @@ export default class ProfilePicture extends React.Component {
-                    src={this.props.src}
+                    className='user-popover__image'
+                    src={profileImg}
+                    height='128'
+                    width='128'
+                    key='user-popover-image'
                 />
-            </span>
-        );
+                );
+            const fullname = Utils.getFullName(this.props.user);
+            if (fullname) {
+                dataContent.push(
+                    <div
+                        data-toggle='tooltip'
+                        title={fullname}
+                        key='user-popover-fullname'
+                    >
+                        <p
+                            className='text-nowrap'
+                        >
+                            {fullname}
+                        </p>
+                    </div>
+                        );
+            }
+            if (global.window.mm_config.ShowEmailAddress === 'true' || UserStore.isSystemAdminForCurrentUser() || this.props.user === UserStore.getCurrentUser()) {
+                dataContent.push(
+                    <div
+                        data-toggle='tooltip'
+                        title={email}
+                        key='user-popover-email'
+                    >
+                        <a
+                            href={'mailto:' + email}
+                            className='text-nowrap text-lowercase user-popover__email'
+                        >
+                            {email}
+                        </a>
+                    </div>
+                );
+            }
+            toRender = (
+                <OverlayTrigger
+                    trigger='click'
+                    placement='right'
+                    rootClose={true}
+                    overlay={
+                        <Popover
+                            title={'@' + this.props.user.username}
+                            id='user-profile-popover'
+                        >
+                            {dataContent}
+                        </Popover>
+                }
+                >
+                    <span className={`status-wrapper ${statusClass}`}>
+                        <img
+                            className='more-modal__image'
+                            width={this.props.width}
+                            height={this.props.width}
+                            src={this.props.src}
+                        />
+                    </span>
+                </OverlayTrigger>
+            );
+        } else {
+            toRender = (
+                <span className={`status-wrapper ${statusClass}`}>
+                    <img
+                        className='more-modal__image'
+                        width={this.props.width}
+                        height={this.props.width}
+                        src={this.props.src}
+                    />
+                </span>
+            );
+        }
+
+        return toRender;
```

Comment:
Instead of populating and returning `toRender`, can you just return the first JSX block within the if statement and have the default JSX block returned when that if statement is not entered?